app/static/js/shared_list.js

```diff
@@ -0,0 +1,36 @@
+var items_num = 1;
+
+document.addEventListener('click', function(e) {
+    if (e.target && e.target.id.includes('btnRemoveItem')){
+        removeItem(e.target.id);
+    }
+    else if (e.target && e.target.id.includes("btnRemoveLabel")) {
+        let parent = document.getElementById(e.target.id).parentElement.id
+        removeItem(parent);
+    }
+    else if (e.target && e.target.id == "btnAddItem"){
+        addItem();
+    }
+});
+
+
+function addItem() {
+    const remove_button = 7;
+    const remove_label = 1;
+    let item = document.getElementById("shared_list_item").cloneNode(true);
+    item.className = "shared_list_item_on"
+    item.childNodes[remove_button].id += String(items_num);
+    item.childNodes[remove_button].childNodes[remove_label].id += String(items_num);
+    items_num++;
+    for (i = 0; i < item.children.length; i++ ) {
```

Comment:
can also use `for (child of item.children)`

Reply:
I'm a newbie in JS, so it's pretty helpful actually. Thanks.